ISAAR.MSolve.Logging/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Remove packages.config file from your pull request since it does not reflect any actual code changes.